server/routerlicious/packages/services-core/src/mongoDatabaseManager.ts

```diff
@@ -31,6 +32,10 @@ export class MongoDatabaseManager implements IDatabaseManager {
 		return this.getCollection<IDocument>(this.documentsCollectionName);
 	}
 
+    public async getCheckpointCollection(): Promise<ICollection<ICheckpoint>> {
```

Comment:
This db manger should have a function of `getCollection`, you can directly use  that if you want to instead of create a new method. Again this dbmanager is poorly implement I would suggestion to not use it

Reply:
Hi Xin, the IDatabaseManager does not include getCollection directly, and abstracts it to have functions related to returning the specific collections, which is why I implemented it this way